### PR TITLE
Show claimed token amount on confirmation screen

### LINKED ISSUE #17: Show claimed amount on claim confirmation screen

### PR DESCRIPTION
**Pull request by weilbith**
Closes #17

### CODE REVIEW

File: src/js/merkle-drop/ClaimFlow.js [hunk 1]
```diff
@@ -328,7 +328,11 @@ function ClaimFlow() {
       return (
         <div className="columns is-centered">
           <div className="column is-three-fifths">
-            <ClaimSuccess txHash={txHash} confirmations={confirmations} />
+            <ClaimSuccess
+              txHash={txHash}
+              confirmations={confirmations}
+              amount={currentClaimAmount}
```
Comment (berndbohmeier) [inline src/js/merkle-drop/ClaimFlow.js]: This shows what our backend calculated what should be claimable, but not what was actually claimed. Please parse the event from the blockchain to be sure to show the actually claimed amount.

Comment (weilbith) [reply src/js/merkle-drop/ClaimFlow.js]: :heavy_check_mark:

File: src/js/merkle-drop/api/web3.js [hunk 2]
```diff
@@ -38,8 +34,46 @@ export async function claimTokens(
   }
 }
 
+export async function getClaimedTokenAmountByReceipt(receipt) {
+  const merkleDropContract = getMerkleDropContract()
+  const eventFilter = {
+    fromBlock: receipt.blockNumber,
```
Comment (berndbohmeier) [inline src/js/merkle-drop/api/web3.js]: we can also set toBlock, cant we?

File: src/js/merkle-drop/api/web3.js [hunk 3]
```diff
@@ -38,8 +34,46 @@ export async function claimTokens(
...
+    transactionHash: receipt.transactionHash,
```
Comment (berndbohmeier) [inline src/js/merkle-drop/api/web3.js]: I did not find in the documentation that you can use that? Where did you find that?

File: src/js/merkle-drop/api/web3.js (hunk 2 above)
Comment (weilbith) [reply src/js/merkle-drop/api/web3.js]: :heavy_check_mark:

File: src/js/merkle-drop/api/web3.js (hunk 3 above)
Comment (weilbith) [reply src/js/merkle-drop/api/web3.js]: To be honest I could not remember anymore, but could not find it myself again. Apparently that gets accepted. It works in the tests, but I don't trust the `web3js` implementation for that. So I removed it and do this part of the filter locally. Having the withdraw event for a single block it should not be much work. By that I took the opportunity and tried to produce more clean code.

Comment (weilbith) [reply src/js/merkle-drop/api/web3.js]: I tested it again with a deployment on _Kovan_.

**Comment by weilbith** [comment]
Sorry, but I'm confused. :confused: 
What refers `Same here, ` to? And as [I stated](https://github.com/trustlines-protocol/www.trustlines.foundation/pull/19#discussion_r372467641) in the conversation above did I test it again with a new contract deployment on _Kovan_. If you like any other tests as well I could do them. But I would love to merge this to the end of this sprint today. :upside_down_face:

**Comment by berndbohmeier** [comment]
Do we have a test page set up?

**Comment by berndbohmeier** [comment]
I meant that we have no test setup and I could not test it

**Pull request merged**